Orange/widgets/data/owtranspose.py

```diff
@@ -45,8 +43,7 @@ def __init__(self):
             gui.indentedBox(
                 box, gui.checkButtonOffsetHint(self.feature_radio.buttons[0])),
             self, "feature_names_column", callback=self._feature_combo_changed,
-            model=self.feature_model, sendSelectedValue=True,
```

Comment:
`sendSelectedValue` is inapplicable when the combo uses a model --- which always returns the object from the model.